frontend/src/components/widgets/NumberInput/styled-components.ts

```diff
@@ -34,7 +47,7 @@ export const StyledInputContainer = styled.div(({ theme }) => ({
 }))
 
 export const StyledInputControls = styled.div({
-  height: "40px",
+  height: "38.39px",
```

Comment:
Hm, this value feels pretty arbitrary to me -- is there a way of computing this from other values or making this feel less like a magic number in some other way?

Reply:
Agreed! Found a way to calculate it automatically [here](https://github.com/streamlit/streamlit/pull/5193/commits/2fcbf0f75de3fe6bb0f058f54186e875ccc6612f)